src/nix/daemon.md

```diff
@@ -8,6 +8,19 @@ R""(
   # nix daemon
   ```
 
+* The daemon does not have native support for listening on a TCP
+  socket, but you can do this using `socat`:
```

Comment:
```suggestion
* The daemon does not have native support for opening on a TCP
  socket to listen to, but you can do this using `socat`:
```

Reply:
Since it does have native support to listen on an already-open file descriptor via the systemd env vars.